zerver/openapi/zulip.yaml

```diff
@@ -4532,18 +4532,17 @@ paths:
 
                                             [calc-full-member]: /api/roles-and-permissions#determining-if-a-user-is-a-full-member
                                     create_multiuse_invite_group:
-                                      type: integer
-                                      description: |
-                                        The ID of the [user group](/api/get-user-groups) whose members are
-                                        allowed to create [reusable invitation
-                                        links](/help/invite-new-users#create-a-reusable-invitation-link)
-                                        to the organization.
-
-                                        This setting can currently only be set to user groups that are
-                                        system groups, except for the system groups named
-                                        `"role:internet"` and `"role:owners"`.
-
-                                        **Changes**: New in Zulip 8.0 (feature level 209).
+                                      allOf:
+                                        - $ref: "#/components/schemas/GroupSettingValue"
+                                        - description: |
+                                            A [group-setting value](/api/group-setting-values) defining the
+                                            set of users who are allowed to create [reusable invitation
+                                            links](/help/invite-new-users#create-a-reusable-invitation-link)
+                                            to the organization.
+
+                                            **Changes**: New in Zulip 8.0 (feature level 209).
+                                            Prior to Zulip 10.0 (feature level 314), this value used to be of
```

Comment:
We always write these changelog entries with most recent change first so readers can stop reading when they have the information they need for recent versions

Reply:
[Fixed.](https://github.com/zulip/zulip/compare/8628c9625b928c5de53d44b20e39d5789829051a..140c4441397f0a1fc0362b86613fad547b7b4b5f)